src/index.js

```diff
@@ -34,3 +34,6 @@ tasks.forEach((task) => {
 
 // Sort tasks by index
 tasks.sort((a, b) => a.index - b.index);
+
+// Local Storage
+localStorage.setItem('tasks', JSON.stringify(tasks));
```

Comment:
- [OPTIONAL] Kindly improve your design 🎨 🖌️  The CSS is created and used to style the list using the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). (Note: The video has more features than the current project, you don't need to style all the features not implemented yet in the video.)